frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.module.css

```diff
@@ -7,3 +7,8 @@
   --mb-color-logo-primary: var(--mb-base-color-gray-30);
   --mb-color-logo-secondary: var(--mb-base-color-gray-10);
 }
+
+.labelMuted:not([data-css-specificity-hack="🐈"]) {
```

Comment:
Use @iethree's approach for ~hacking~increasing CSS specificity.

Reply:
(I wonder how many unique emojis we will end up consuming if we keep doing this without repeating the same emoji... Hopefully not too many 😆)

Reply:
It seems like a good idea to follow our CEO (Chief Emoji Officer) style, so we should ensure we never use the same emoji twice. I think it's going to be awhile until we have to repeat one :)

Reply:
nit: are we sure we need this hack?
Label is a styled component, we could add add a mantine-like prop to it for the colors. If we migrate out of styled components that should probably become a component from metabase/ui anyway

Reply:
> nit: are we sure we need this hack?

Yes, my class lost to style from emotion :(

Reply:
I meant we could just pass a variant/color props to it.
I know we want to eventually remove emotion, but Label is a styled component and mixing styled and css modules seems to lead to even more issues

Reply:
good point. Maybe it's not that hard to convert the Label to use CSS module then. If that works we won't need this hack. Passing a single color to components is fine, but it's becoming more complicated when we want to pass multiple colors.

In this case, we want to use CSS module because a single class can set multiple CSS custom properties at once. We can pass this via `style` though we need to satisfy TypeScript by doing something more, I don't exactly remember the detail. But we would need to pass something like this.
```ts
style={{ '--mb-my-color', '#aaa', '--mb-my-other-color', '#bbb' }} // this works, but TS would complain
```

I don't know how much time I'd be working on this on the offsite, but if I have time I'll try to migrate away from emotion.

Reply:
@npretto I've modified some of the code to just not use CSS or the styled components, I've moved most of everything to Mantine components so we don't have to deal with it 😄